inode.c

```diff
@@ -63,10 +63,18 @@ struct inode *simplefs_iget(struct super_block *sb, unsigned long ino)
     inode->i_ctime.tv_nsec = 0;
 #endif
 
+#if LINUX_VERSION_CODE >= KERNEL_VERSION(6, 7, 0)
+    inode->__i_atime.tv_sec = (time64_t) le32_to_cpu(cinode->i_atime);
```

Comment:
The accessors should be used instead of modifying the members directly.

Reply:
I've been using the accessors in the newest commit!